tests/e2e/Output_Stream/.gitignore

```diff
@@ -0,0 +1 @@
+/stderr.log
```

Comment:
Can we use `infection.log` as ignored globally in `tests/e2e/.gitignore`?

Reply:
I wasn't to use a non-standard filename for this test, to ensure that we weren't getting `infection.log` written by something else (other than the output redirect). Just to make the test a little more robust against regressions.

Happy to change this to `infection.log`, delete the others, and add a more general `.gitignore` if you prefer?

Reply:
I don't think "something else" can write that file, since it's being removed right before the infection run, and no other loggers are configured. So I would remove it to make it a little bit more simple and consistent